src/mudlet.cpp

```diff
@@ -2375,6 +2375,14 @@ void mudlet::setEditorTextoptions( const bool isTabsAndSpacesToBeShown, const bo
     emit signal_editorTextOptionsChanged( mEditorTextOptions );
 }
 
+void mudlet::setEchoLuaErrors( const bool isEchoLuaErrorsEnabled )
```

Comment:
Are you saying just getting rid of the setEchoLuaErrors function and manually editing it via like pHost->mEchoLuaErrors = true/false? I thought about doing that, but was just sort of emulating the code I saw around it, since I'm coming into this with a less than thorough understanding of the interactions between the various parts of Mudlet. If it would work though, I could do that and simplify the code a bit.